src/components/CoursesTabs.js

```diff
@@ -1,114 +1,127 @@
-import React, { useState } from 'react'
-import Calendar from 'reactjs-calendar'
-import Courses from './Courses'
-import Grades from './Grades'
-import PropTypes from 'prop-types'
-import TermSelect from './TermSelect'
-import { useSelector } from 'react-redux'
-import { useTranslation } from 'react-i18next';
+import React, { useState, useEffect } from 'react'
 
 import AppBar from '@material-ui/core/AppBar'
 import Assignment from '@material-ui/icons/Assignment'
+import BuyBooks from './BuyBooks'
+import Calendar from './Calendar'
+import Courses from './Courses'
 import Event from '@material-ui/icons/Event'
+import Grades from './Grades'
 import Paper from '@material-ui/core/Paper'
+import PrintCourses from './PrintCourses'
+import PropTypes from 'prop-types'
 import Spellcheck from '@material-ui/icons/Spellcheck'
 import Tab from '@material-ui/core/Tab'
 import Tabs from '@material-ui/core/Tabs'
+import TermSelect from './TermSelect'
 import Toolbar from '@material-ui/core/Toolbar'
-import { makeStyles } from '@material-ui/styles';
+import useMediaQuery from '@material-ui/core/useMediaQuery'
+import { fetch_events } from '../actions/eventsActions'
+import { fetch_selected_courses } from '../actions/termActions'
+import { makeStyles } from '@material-ui/styles'
+import { useDispatch } from 'react-redux'
+import { useSelector } from 'react-redux'
+import { useTheme } from '@material-ui/core'
 
-const TabContainer = props => (
+const TabContainer = (props) => (
   <div style={{ padding: 20 }}>{props.children}</div>
 )
 
 TabContainer.propTypes = {
-  children: PropTypes.node.isRequired
+  children: PropTypes.node.isRequired,
 }
 
-const useStyles = makeStyles(theme => ({
+const useStyles = makeStyles((theme) => ({
   root: {
-    minHeight: 0
+    minHeight: 0,
   },
-
   tab: {
     '@media (min-width: 1024px)': {
-      minWidth: 72
-    }
-  },
-
-  inner: {
-    marginTop: 30,
-    width: '100%'
+      minWidth: 72,
+    },
   },
-  appBar: {
-    backgroundColor: theme.palette.primary.main,
-    color: theme.palette.primary.contrastText
-  },
-
   flex: {
-    flex: 1
+    flex: 1,
   },
-
   button: {
-    color: '#FFFFFF'
+    color: '#FFFFFF',
   },
   bar: {
-    flexDirection: 'column'
-  }
+    flexDirection: 'column',
+  },
+  btnContainer: {
+    display: 'flex',
+  },
 }))
 
-export default function CoursesTabs(props) {
+export default function CoursesTabs() {
   const [value, setValue] = useState(0)
-  const term_bounds = useSelector(state => state.terms.term_bounds)
-  const { mobile } = props
-  const { t } = useTranslation()
+  const courses = useSelector((state) => state.courses)
+  const books = useSelector((state) => state.books)
+  const selected_term = useSelector((state) => state.selected_term)
   const classes = useStyles()
+  const theme = useTheme()
+  const mobile = useMediaQuery(theme.breakpoints.down('xs'))
```

Comment:
This is such a better way of determining this.

Reply:
You can thank @amgoodfellow for that one